src/hotspot/share/gc/shared/collectedHeap.hpp

```diff
@@ -141,6 +142,10 @@ class CollectedHeap : public CHeapObj<mtGC> {
   PerfStringVariable* _perf_gc_cause;
   PerfStringVariable* _perf_gc_lastcause;
 
+  // Perf counters for CPU time of parallel GC threads. Defined here in order to
+  // be reused for all collectors.
+  PerfVariable* _perf_parallel_gc_threads_cpu_time;
```

Comment:
If this is intended to be reused for other GCs then rename to something more generic like `_perf_gc_threads_cpu_time`.

Reply:
The intention is for parallel GC worker threads. Not for all GC threads, and not to be confused with threads for `ParallelGC`.
Perhaps `_perf_parallel_worker_threads_cpu_time` is better?